src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -83,125 +73,103 @@ Session replay minimizes impact on your app's performance with:
+</Callout>
 
+New Relic takes your data privacy seriously. The default privacy setting masks everything by replacing all text with asterisks. Here's an example of how a replay looks with default settings:
 
+<img
+    title="Example of default privacy settings"
+    alt="Example of default privacy settings"
+    src={browserTextMaskingExample}
+/>
 
-      <CollapserGroup>
-      <Collapser
-      id="mask-user-data"
-      title="How to mask user data"
-  >
-      If you want to customize exactly what is masked with asterisks, you have a few options:
+If you need to change the default privacy settings, you can choose specific classes and attributes to:
 
-        * On the <DoNotTranslate>**Application settings**</DoNotTranslate> page, add your own CSS selectors or check the masking boxes.
+* Mask user input
+* Block site content
+* Ignore user input
 
-                <img
-                    title="Mask settings"
-                    alt="Screenshot of mask settings in the ui"
-                    src={browserMaskSettings}
-                />
-        * Add our CSS class `nr-mask` or attribute `data-nr-mask` to your webpage HTML.
-        For example, if you want to mask user-entered email address, add `nr-mask` to your email class attribute in `&lt;div&gt;`:
+### Mask user input [#mask-input]
 
-            ```html
-            <div class="email-input nr-mask">
-              <input type="email" name="email" placeholder="Enter your email address">
-            </div>
-            ```
+You can mask user input, which means the input will be replaced with asterisks (*). For example, a user's login information will render as asterisks in session replay.
 
-        These options use an asterisk (*) to mask all text in that element, hiding the actual text but revealing the number of characters entered. However, asterisks are not valid numbers, so masking number type-specific fields such as telephone or credit card numbers will result in a blank field in a replay.
+To specify which input should be masked, you have a few options:
 
-  </Collapser>
-  </CollapserGroup>
+  * On the <DoNotTranslate>**Application settings**</DoNotTranslate> page, add your own CSS selectors or check the masking boxes. Note that using mask selectors will **not** mask user input, so if you need to hide user input, we recommend using block selectors. 
 
-* <DoNotTranslate>**Block**</DoNotTranslate> means content will be replaced with an empty placeholder. For example, if you have a video playing on your About Us webpage and don’t want it captured, you could block the class that contains that video.
+          <img
+              title="Mask settings"
+              alt="Screenshot of mask settings in the ui"
+              src={browserMaskSettings}
+          />
+  * Add our CSS class `nr-mask` or attribute `data-nr-mask` to your webpage HTML.
+  For example, if you want to mask user-entered email address, add `nr-mask` to your email class attribute in `<div>`:
```

Comment:
You cannot mask user-entered anything. You would use the User input masking options or the block selectors.

Reply:
@jomle What's an example of something (non-user input) customers would want to mask?

Reply:
@jomle Can you provide another example (the html sample below) of what they would want to mask?